src/structure/Astar.lua

```diff
@@ -0,0 +1,98 @@
+local class = require 'lib/30log'
+local Astar = class('Astar',{
+	indexMap = nil
+})
+
+COST_ESTIMATE = 1
+INF = 1/0
```

Comment:
Is there not a math.infinity function in LUA? I recommend math.huge http://www.lua.org/manual/5.1/manual.html#pdf-math.huge